analytics/src/test/java/com/segment/analytics/CryptoTest.kt

```diff
@@ -0,0 +1,40 @@
+package com.segment.analytics
+
+import okio.Buffer
+import okio.ByteString
+import okio.Okio
+import org.assertj.core.api.Assertions
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.annotation.Config
+import java.io.IOException
+
+@RunWith(RobolectricTestRunner::class)
+@Config(manifest = Config.NONE)
+class CryptoTest {
+
+  @Throws(IOException::class)
+  @Test
+  fun noneCryptoWrite() {
+    val crypto: Crypto = Crypto.none()
+    val foo: ByteString = ByteString.encodeUtf8("foo")
+    val buffer = Buffer()
+
+    foo.write(crypto.encrypt(buffer.outputStream()))
+
+    Assertions.assertThat(buffer.readByteString()).isEqualTo(foo)
+  }
+
+  @Test
+  fun cryptoRead() {
+    val crypto: Crypto = Crypto.none()
+    val foo: ByteString = ByteString.encodeUtf8("foo")
+    val buffer = Buffer()
+
+    buffer.write(foo)
+
+    Assertions.assertThat(Okio.buffer(Okio.source(crypto.decrypt(buffer.inputStream()))).readByteString())
```

Comment:
is there a way to better format this code? feels a bit unreadable atm. Im not sure what kotlin usually recommends